components/primitives/Form/Input.tsx

```diff
@@ -32,7 +34,6 @@ const Input = React.forwardRef<HTMLInputElement, InputProps>(
               viewBox="0 0 24 24"
               strokeWidth={1.5}
               stroke="currentColor"
-              className="w-5 h-5 text-abeg-neutral-50 absolute left-[calc(100%-20px)] top-1/2 -translate-x-1/2 -translate-y-1/2"
```

Comment:
Can you please give a reason why you deleted this line? if there was a replacement, can you please point that out?

Reply:
Yeah I basically said he should move it to the button when I wrapped it in a button it broke the styling a bit I just said he should move the styles to the button element itself instead of creating another PR